app/controllers/ansible_repository_controller.rb

```diff
@@ -77,8 +77,12 @@ def display_playbooks
   end
 
   def repository_refresh
-    # Targeted refresh for embedded ansible hasn't been implemented yet
-    embedded_ansible_refresh
+    assert_privileges("embedded_configuration_script_source_refresh")
+    checked = find_checked_items
+    checked[0] = params[:id] if checked.blank? && params[:id]
+    objects = AnsibleRepositoryController.model.find(checked)
```

Comment:
@mzazrivec while debugging https://bugzilla.redhat.com/show_bug.cgi?id=1442170 I added this line here

```ruby
$log.info("durandom: #{checked}")
```

and it logged this for 1 checked item and 2 checked items

```
[root@dhcp-8-198-79 vmdb]# tail -f log/evm.log | grep durandom
[----] I, [2017-04-27T03:05:31.539498 #34572:1772dc8]  INFO -- : durandom: []
[----] I, [2017-04-27T03:06:04.361611 #34581:17729a4]  INFO -- : durandom: []
```

looks like `find_checked_items` does not work for the repo list view?